analysis/src/topEventSelectionDL.cc

```diff
@@ -41,9 +41,16 @@ int topEventSelectionDL::EventSelection() {
   auto muons = muonSelection();
   auto elecs = elecSelection();
 
+<<<<<<< Updated upstream
```

Comment:
You need to fix it when you have a merge error, choose from "<<<" to "===" or "===" to ">>>", or merge them together by hand.

Reply:
Oh, I missed them lines...
I fixed, Ian.